src/@next/DataTable/TableCell.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { TableTypography } from './DataTable';
+import { StyledTableCell } from './DataTableStyle';
+
+export interface TableCellProps
+  extends React.TdHTMLAttributes<HTMLTableColElement> {
+  children: React.ReactNode;
+}
+
+export const TableCell = ({ align, children }: TableCellProps) => {
+  return (
+    <StyledTableCell data-text-align={align}>
+      <div>
```

Comment:
do we still need the extra div?

Reply:
yes, since anything can be inside the cell (not just text) we need this wrapper to align the content

Reply:
Oh I see what you meant, the Typography is already a div, will remove that